modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -13,6 +13,95 @@
+Fulltext search utilizes Apache Tika for more advanced content extraction. Apache Tika ensures lightning-fast search results, delivering near-instantaneous responses to your queries. Whether you have a large document repository or a vast number of files, full text search will provide you with swift and accurate results, enabling you to access the information you need without delay.
+With Apache Tika you can search the contents of the filetypes: PDF, DOCX, XLSX, PPTX, ODF, HTML, XML, EPUB, RTF, Tar, RAR, AR, CPIO, Zip, 7Zip, Gzip, BZip2 and many more ...
+Read more: https://doc.owncloud.com/ocis/next/deployment/services/s-list/search.html#tika-extractor
+
+=== Space Templates
+Applying a template is a breeze. When creating a new project Space, simply rightclick on a folder of your choice, select "Create Space from selection" and the entire folder structure, complete with subfolders and files, will be instantly generated for a project Space. Save time and ensure consistency by eliminating the need to manually create folders and subfolders every time you start a new project or onboard a new team. Space templates ensure a consistent folder structure across projects providing a standardized, efficient, and customizable approach to organizing your documents. Space Templates are only available to users with the permission to create Spaces.
+
+=== Custom user roles
+Infinite Scale ships with the default user roles Admin, Space Admin, User and Guest. Each role has different permissions. For example, only Space Admins can create Spaces, but people with the role "User" cannot create a Space. With custom roles, you can now configure roles that fit the needs of your organization. For example, if you need custom roles for your school, you can configure the roles "Teacher" and "Pupil", and don't give pupils the permission to create public links. If you are interested in creating custom roles, reach out for us on talk.owncloud.com
+
+=== File Versions: Design overhaul
+File versions received a design overhaul so that you can go back easily in the version history of a file. With the new design of the file versions feature, you now have even easier access to a comprehensive version history for each file. Every time a file is modified or updated, a new version is created and stored, preserving a complete timeline of changes. This allows you to track and revert to previous versions as needed, ensuring data integrity and eliminating the risk of accidental data loss.
+
+=== Administration Settings
+This update brings a range of functions to streamline user, group, and space administration, enhancing the overall management experience. Let's explore the exciting new capabilities:
+==== Users Administration:
+- Filter Chips for Groups and Role: You can now conveniently filter users based on groups and roles, making it easier to locate and manage specific user segments efficiently.
```

Comment:
Chips? No idea what these are supposed to be.

Reply:
enjoy ;-) https://m2.material.io/components/chips#filter-chips

Reply:
> enjoy ;-) https://m2.material.io/components/chips#filter-chips

Ah, I see. That makes sense. Thank you! :-)